api/llm_inference/v1beta1/lm_inference_utils.go

```diff
@@ -0,0 +1,58 @@
+package llm_inference
+
+import (
+	"github.com/scaleway/scaleway-sdk-go/internal/errors"
+	"github.com/scaleway/scaleway-sdk-go/internal/async"
+	"time"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	defaultRetryInterval = 2 * time.Second
```

Comment:
Question: how was decided the default retry interval, considering we have 15s in rdb?